### PR TITLE
Approach Logger

### PR DESCRIPTION
**Pull request by anghelitalo**
Added approach to logs.

### CODE REVIEW

**Review by AaronClaros — APPROVED** [review]
approved

File: src/util/logger.py [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+        # set a format which is simpler for console use
+        formatter = logging.Formatter('%(name)-12s: %(levelname)-8s %(message)s')
+        # tell the handler to use this format
+        console.setFormatter(formatter)
+        # add the handler to the root logger
+        logging.getLogger('').addHandler(console)
+
+    """Return a message type debug.
+    @msg receive a text.
+    """
```
Comment (sperBruno) [inline src/util/logger.py]: move all documentation inside the methods

File: config.ini [hunk 2]
```diff
@@ -6,10 +6,12 @@ apiToken=f89634fa55d2cb80fee84fd0d6a9fd2a
...
 user2=adrinrojas1
-password2=test12345
+password2=
 
 user3=kevinherrera2
-password3=70723844
\ No newline at end of file
+password3=
+
+log_path = c:\\temp\\pivotal.log
```
Comment (sperBruno) [inline config.ini]: logger path should point to the project root

File: src/util/logger.py [hunk 3]
```diff
@@ -0,0 +1,62 @@
+"""
+Section imports.
+"""
+from src.util.ReadCfg import ReadCfg
+import logging
+
+"""
+This class is to logs.
```
Comment (sperBruno) [inline src/util/logger.py]: move documentation inside class

**Pull request merged**